components/Common/Search/index.module.css

```
@@ -29,6 +29,8 @@
     absolute
     right-2
     ml-2
+    transition-opacity
+    duration-500
```

```suggestion
    motion-safe:transition-opacity
    motion-safe:duration-200
```

It may be better to use `safe-motion` for users who are sensitive to this type of interaction. IMO 500 ms is a little bit long 😄

---

Thank you for your feedback! I agree with you on the importance of accommodating users who are sensitive to motion. That's why I've implemented the motion-safe utility to ensure that transitions only occur if the user has not specified a preference for reduced motion. Regarding the duration, I've adjusted it to 100ms to make the transition quicker while still providing a noticeable effect. 😊